docs/APIs/io/AnalogOut.md

```diff
@@ -5,7 +5,7 @@ Use the AnalogOut interface to set the voltage of an analog output pin in the ra
 **Tips:**
 
 * Not all pins are capable of being AnalogOut, so check the documentation.</br>
-* For more information on what it takes to convert an digital value to its analog representation see [http://en.wikipedia.org/wiki/Digital-to-analog_converter](http://en.wikipedia.org/wiki/Digital-to-analog_converter).   
+* For more information on what it takes to convert an digital value to its analog representation, see [http://en.wikipedia.org/wiki/Digital-to-analog_converter](http://en.wikipedia.org/wiki/Digital-to-analog_converter).   
```

Comment:
I would maybe say "For more information about converting a digital value".
Either way it should be "a digital" (this feels like a copy/paste from "an analogue")